dbms/src/TiDB/Decode/JsonBinary.h

```diff
@@ -193,6 +193,9 @@ class JsonBinary
 
     static void assertJsonDepth(UInt64 depth);
 
+    static bool isJSONContainsPathAll(const std::string_view & type);
```

Comment:
If it is used for json_contains_path only, prefer to move to FunctionJson.h instead.

Reply:
ok, moved.